packages/drawer/src/views/DrawerView.tsx

```diff
@@ -46,7 +46,7 @@ const getDefaultDrawerWidth = ({
   /*
    * Default drawer width is screen width - header height
    * with a max width of 280 on mobile and 320 on tablet
```

Comment:
It looks like the default values are no longer listed on the [linked page](https://material.io/components/navigation-drawer) nor on the [Android specific page](https://material.io/develop/android/components/navigation-view). 

Let me know if I should removed this link from the comment.